### PR TITLE
feat(dropdown-links): add new component

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-820

Capture d’écran 2023-09-28 à 16 38 49

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> There are two hard problems in computer science: naming things, cache invalidation and off-by-1 errors

File: libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  Truncate,
+} from '@qovery/shared/ui'
+import { plural } from '@qovery/shared/util-js'
+
+export interface ServiceLinksDropdownProps {
+  links?: Link[]
+}
+
+// TODO: Update props and using React Query
+export function ServiceLinksDropdown({ links }: ServiceLinksDropdownProps) {
```
Comment (ctjhoa) [inline libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx:19]: Not sure about the naming here.
IMO it should be named after the functional behavior. (ServicePreviewLinks ?)
And if not, it should be `ServiceLinksPopover`.
We can talk about it.

File: libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          </Skeleton>
+        </div>
+      </Popover.Trigger>
+      <Popover.Content
+        side="bottom"
+        className="p-2 text-neutral-350 text-sm relative right-4 border-transparent"
+        style={{ width: 280 }}
+      >
+        <div className="p-2 flex justify-between items-center">
+          <h6 className="text-neutral-350 font-medium">
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx:43]: mmh I based it on your integration for Popover, but I can add a `<p>` 

Capture d’écran 2023-09-29 à 13 53 20

File: libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx:19]: `ServiceLinksPopover` looks better yes, and it's a Popover, not a dropdown as well

File: libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            {links?.length ?? 0} {plural(links?.length ?? 0, 'link')} attached
+          </h6>
+        </div>
+        <ul>
+          {links?.map((link: Link) => (
+            <li key={link.url} className="flex p-2">
+              <CopyToClipboard className="text-brand-500 hover:text-brand-600 mr-2" content={link.url ?? ''} />
+              <a
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx]: We can yes, even if the style is replaced

File: libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx]: https://github.com/Qovery/console/pull/859/commits/3fd725f5aa63a38744861a8930a1106b9ba56f74

File: libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx]: https://github.com/Qovery/console/pull/859/commits/3fd725f5aa63a38744861a8930a1106b9ba56f74

**Pull request merged**